posthog/hogql_queries/error_tracking_query_runner.py

```diff
@@ -80,15 +146,72 @@ def calculate(self):
             filters=HogQLFilters(
                 dateRange=self.query.dateRange,
                 filterTestAccounts=self.query.filterTestAccounts,
-                properties=properties,
+                properties=self.properties,
             ),
         )
 
+        columns: list[str] = query_result.columns or []
+        results = self.results(columns, query_result.results)
+
         return ErrorTrackingQueryResponse(
-            columns=query_result.columns,
-            results=query_result.results,
+            columns=columns,
+            results=results,
             timings=query_result.timings,
             hogql=query_result.hogql,
             modifiers=self.modifiers,
             **self.paginator.response_params(),
         )
+
+    def results(self, columns, query_results):
+        mapped_results = [dict(zip(columns, value)) for value in query_results]
+        results = []
+        for result_dict in mapped_results:
+            fingerprint = self.query.fingerprint if self.query.fingerprint else result_dict["fingerprint"]
+            group = self.group_or_default(fingerprint)
+            events: list = []
+            if "events" in result_dict:
+                events = [
+                    {"uuid": str(e[0]), "properties": e[1], "timestamp": e[2]} for e in result_dict.get("events", [])
+                ]
+            results.append(group | result_dict | {"events": events})
+        return results
+
+    @property
+    def order_by(self):
+        return (
+            [
+                ast.OrderExpr(
+                    expr=ast.Field(chain=[self.query.order]),
+                    order="ASC" if self.query.order == "first_seen" else "DESC",
+                )
+            ]
+            if self.query.order
+            else None
+        )
+
+    @cached_property
+    def properties(self):
+        return self.query.filterGroup.values[0].values if self.query.filterGroup else None
+
+    def where_exprs(self):
+        return []
+
+    def group_or_default(self, fingerprint):
+        return self.error_tracking_groups.get(
+            fingerprint,
+            {
+                "fingerprint": fingerprint,
+                "assignee": None,
+                "merged_fingerprints": [],
+                "status": str(ErrorTrackingGroup.Status.ACTIVE),
+            },
+        )
+
+    @cached_property
+    def error_tracking_groups(self):
```

Comment:
Loads all active groups which again might be a performance constraint but should be ok for now given groups are lazily created when they don't match [the default](https://github.com/PostHog/posthog/pull/23837/files#diff-36c997637a73690a10ec448c3ea63835299dd76267681a1be67dc9332ff6bd3dR202-R207): 
```
{
    "fingerprint": fingerprint,
    "assignee": None,
    "merged_fingerprints": [],
    "status": str(ErrorTrackingGroup.Status.ACTIVE),
}
```